### DISCUSSION TITLE
GitHub Desktop for Windows Authentication failed

### DISCUSSION THREAD

**Discussion by mferino** [community/community#22560] [New to GitHub]
<p>Hi, </p>
<p>I’m a GitHub noob and I’m needing to use GitHub Desktop for an EdX course that I’m taking.  A big part of it is to clone this **<a href="https://github.com/UBCx-Software-Construction/PracticeProblems." rel="nofollow noopener">https://github.com/UBCx-Software-Construction/PracticeProblems.  </a>**Every time I go to clone I get an Authenticaiton failed error and it asks my username and password.  I certainly have the correct credentials and I’ve tried a few other fixes such as logging out and in, deleting the github credentials in Windows and removing the authorization between github and github desktop then reactivating it.  Any help would be very much appreciated so I can get started on this course!</p>

**Comment by steveward** [comment, marked as answer]
<p><a class="mention" href="/phwumph">@phwumph</a> sorry to hear you are having trouble. It looks like the repository you are attempting to clone does not exist on GitHub – is this possibly the repository you are wanting to clone?</p>
<p><a href="https://github.com/UBCx-Software-Construction/TPD-practice-projects" rel="nofollow noopener">https://github.com/UBCx-Software-Construction/TPD-practice-projects</a></p>
<p>You can quickly clone this repository by clicking the green <code>Clone or download</code> button on the repository’s page and selecting <code>Open in Desktop</code>. This will launch GitHub Desktop and allow you to choose a location to clone to.</p>
<p>A quick note about this error in case anyone else runs across this – when you attempt to clone a repository that does not exist in GitHub Desktop it will throw an <code>authentication failed</code> error, which is confusing since it’s not actually an authentication issue. We’re working to improve these error messages to make it clearer what the actual issue is (in this case the fact that the repository does not exist).</p>
<p>Let me know if you have any issues cloning the correct repository. </p>
